BackendApartmentReservation/BackendApartmentReservation/Managers/TripManager.cs

```diff
@@ -0,0 +1,40 @@
+﻿using BackendApartmentReservation.Database.Entities;
+using BackendApartmentReservation.DataContracts.DataTransferObjects.Requests;
+using BackendApartmentReservation.Repositories;
+using BackendApartmentReservation.Repositories.Checklist;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.Managers
+{
+    public class TripManager : ITripManager
+    {
+        private readonly ITripRepository _tripRepository;
+        private readonly IChecklistRepository _checklistRepository;
+
+        public TripManager(ITripRepository tripRepository, IChecklistRepository checklistRepository)
+        {
+            _tripRepository = tripRepository;
+            _checklistRepository = checklistRepository;
+        }
+
+        public async Task<string> CreateBasicTrip(CreateTripRequest tripRequest)
+        {
+            var trip = await _tripRepository.CreateTrip(tripRequest);
+
+            foreach (var group in trip.Groups)
+            {
+                group.Employees.ForEach(async e =>
+                {
+                    var checklist = new DbEmployeeAmenitiesChecklist
```

Comment:
I believe I have added checklist manager method to create a basic checklist for a person

Reply:
I could use that, but I didn't want to create a dependency on other manager

Reply:
The thing is that it does more than just create empty checklist. So even though I am not a fan of using managers in other managers too, its better to do that in this case